sherpa/ui/utils.py

```diff
@@ -973,6 +987,25 @@ def clean(self):
             'source_component': sherpa.image.ComponentSourceImage()
         }
 
+    def set_plot_backend(self, backend):
+        """Change the plot backend.
+
+        This will reset any plot structures, such as that returned by
+        get_data_plot.
+
+        Parameters
+        ----------
+        backend : str
+            The name of the plot backend.
+
+        """
+
+        set_backend(backend)
+
+        # Re-create all the plot objects
+        self._set_plot_types()
+        self._set_contour_types()
+
```

Comment:
We need to add (guessing on the number of spaces)

```
        set_plot_backend
````

to

- docs/ui/sherpa_ui.rst
- docs/ui/sherpa_astro_ui.rst

Reply:
I'm so used to https://github.com/astropy/sphinx-automodapi that I always forget that I do need to add things manually in sherpa. Well, maybe there is another sphinx related PR in my future...

Reply:
There is a reason I didn't use this to begin with, which is some combination of

- didn't know this existed
- it didn't exist at the time I started writing the sphinx docs
- I had trouble getting it (or something equivalent) to work
- I am doing something against the grain with some of the modules and splitting them up, so we have different parts listed in different places
  - this is probably a bad idea but I have no energy to "fix" this

Reply:
It's improved a lot over the years, so just because it did not work in the past, does not mean it won't work now. No hurry either way, just explaining why I forgot about adding those manually to the docs.